examples/swap_networks.py

```diff
@@ -0,0 +1,83 @@
+"""Demonstrates swap networks."""
```

Comment:
Describe in a lot more detail the purpose and mechanism of the example. Users are going to be looking at this file with zero context.

Reply:
I added some explanation. Let me know if you'd like more.

Is there a style convention for example docstrings? What I wrote is a weird mix of python, unicode, and (unrendered) tex. Rendered latex would be nicest, but right now the example files are shown in RTD verbatim.